packages/video_player_avplay/tizen/src/media_player.cc

```diff
@@ -52,7 +53,22 @@ MediaPlayer::MediaPlayer(flutter::BinaryMessenger *messenger,
   media_player_proxy_ = std::make_unique<MediaPlayerProxy>();
 }
 
-MediaPlayer::~MediaPlayer() { Dispose(); }
+MediaPlayer::~MediaPlayer() {
+  if (player_) {
+    player_stop(player_);
+    player_unprepare(player_);
+    player_unset_buffering_cb(player_);
+    player_unset_completed_cb(player_);
+    player_unset_interrupted_cb(player_);
+    player_unset_error_cb(player_);
+    player_unset_subtitle_updated_cb(player_);
+    player_destroy(player_);
+    player_ = nullptr;
+  }
```

Comment:
How about making `PlusPlayer`'s destructor code style and error log the same?
(`drm_manager_` and `player_` seem to have the same life cycle, but the processing results vary if player_ is null in `MediaPlayer` class.)

Reply:
Fixed